### PR TITLE
Release/azure identity 1.8.0

### PR DESCRIPTION
**Pull request by billwert**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by vcolin7 — APPROVED** [review]
We should also update the README to mention the current version to use in people's POMs is `1.8.0`. Besides that things LGTM :)

Approving to unblock as well.

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 1]
```diff
@@ -1,22 +1,15 @@
 # Release History
 
-## 1.8.0-beta.2 (Unreleased)
+## 1.8.0 (2023-02-03)
 
 ### Features Added
+- Added support to configure `clientOptions`, `httpLogOptions`, `retryPolicy`, `retryOptions` and `addPolicy` on Identity credentials.
```
Comment (g2vinay) [inline sdk/identity/azure-identity/CHANGELOG.md:6]: I believe we discussed calling out in changelogs which features are being GA'ed from beta series.
adding a subheader something along the lines of "Features being GA'ed from v1.8.0-beta.1"

File: sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientOptions.java [hunk 2]
```diff
@@ -104,6 +102,11 @@ public IdentityClientOptions setAuthorityHost(String authorityHost) {
         return this;
     }
 
+    /**
+     * Disables authority validation when required for Azure Active Directory token endpoint.
+     * @return IdentityClientOptions
+     */
+
```
Comment (g2vinay) [inline sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientOptions.java:109]: This javadoc seems in a floating location.

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 3]
```diff
@@ -1,22 +1,17 @@
...
 ### Features Added
 
-- [[#32527]](https://github.com/Azure/azure-sdk-for-java/pull/32527) Added Azure Developer CLI Credential.
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+#### Features Generally Available from v1.8.0-beta1
```
Comment (vcolin7) [inline sdk/identity/azure-identity/CHANGELOG.md:7]: Just a suggestion, what you wrote is also fine.

```suggestion
#### From version 1.8.0-beta.1
```

**Review by vcolin7 — APPROVED** [review]

File: sdk/identity/azure-identity/CHANGELOG.md (hunk 3 above)
Comment (billwert) [reply sdk/identity/azure-identity/CHANGELOG.md:7]: we've gotten some feedback on the Identity readmes from users that they want clarity about what is now GA so I want to use that language specifically.

**Pull request merged**